### ISSUE TITLE
produce lowPtGsfEle slimmed collections for the default miniAOD

### ISSUE THREAD

**Issue by slava77** [cms-sw/cmssw#25884]
This is a follow up to #25753 .
During the review of this PR the new slimmed low-pt electron collections were added to the miniAOD only for bParking era/modifier due to a significant size increase for the default setup.
This issue should aim for a working low-pt electron setup in the default miniAOD to make use of the products created in RECO and saved in AOD.

**Comment by slava77** [comment]
assign reconstruction,xpog

@Sam-Harper @lsoffi @bainbrid @mverzett 
I consider this as a somewhat low priority/best-effort case and perhaps to be implemented with some real feedback from the bParking data experience.

**Comment by slava77** [comment]
@jainshilpi @lsoffi  @afiqaize @SohamBhattacharya 
was there any progress on this topic?

**Comment by jainshilpi** [comment]
The work on finalizing the regression and ID MVA is ongoing. Once that is done (~a months time), it can be put in miniAOD.

**Comment by slava77** [comment]
@jainshilpi @lsoffi @afiqaize @SohamBhattacharya
please update if there was some recent progress on this topic.
Thank you.

**Comment by bainbrid** [comment]
Hi @slava77, I'm in touch with @jainshilpi and @lsoffi to figure out what's needed. We'll get back to you ASAP.

**Comment by slava77** [comment]
> @jainshilpi @lsoffi @afiqaize @SohamBhattacharya
> please update if there was some recent progress on this topic.
> Thank you.

ping.
AFAIK, some work has been done recently; it would be nice to know (updated) expected time to delivery.
Thank you.

**Comment by jainshilpi** [comment]
Talking to Chiara, in ~2 weeks, they expect it to be done.

**Comment by slava77** [comment]
+1

provided with #31220 (11_2_0) for a default miniAOD and #32372 for UL re-miniAOD in 10_6_X since CMSSW_10_6_20
